gapic-common/gapic-common.gemspec

```diff
@@ -36,15 +36,15 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.5"
 
   spec.add_dependency "faraday", "~> 1.3"
-  spec.add_dependency "googleapis-common-protos", ">= 1.3.11", "< 2.0"
-  spec.add_dependency "googleapis-common-protos-types", ">= 1.0.6", "< 2.0"
-  spec.add_dependency "googleauth", "~> 0.15", ">= 0.15.1"
-  spec.add_dependency "google-protobuf", "~> 3.15", ">= 3.15.2"
```

Comment:
wasn't there sometine specific in 3.15.2 that we shouldn't loose reverting to 3.14?

Reply:
We did that because early versions of 3.15.x had some regressions that made them unusable in some circumstances. Those regressions weren't present in 3.14.

Originally we updated the minimum dependency to 3.15 because it was the first version to support Ruby 3. However, we're getting feedback that some customers want to remain on 3.14 for the time being because of stability. Moving the minimum back to 3.14 will support those customers as long as they are on Ruby 2.x.